modules/javafx.graphics/src/main/java/com/sun/glass/ui/View.java

```diff
@@ -963,7 +968,12 @@ protected void notifyScroll(int x, int y, int xAbs, int yAbs,
     }
 
     protected void notifyKey(int type, int keyCode, char[] keyChars, int modifiers) {
-        handleKeyEvent(System.nanoTime(), type, keyCode, keyChars, modifiers);
+        handleKeyEvent(System.nanoTime(), type, keyCode, keyChars, modifiers, -1);
+    }
+
+    // Returns true iff event was consumed
```

Comment:
There are some places where it's written "iff", I'm not sure if it's intentional or a typo.

Reply:
iff is an abbreviation for "if and only if". It's a bit formal but I noticed it was used in other internal API comments and JavaDocs. I think I removed all of these when I resubmitted this code as part of #1126.